packages/design-system/src/Input/Input.tsx

```diff
@@ -46,6 +46,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
       type = "text",
       UNSAFE_height,
       UNSAFE_width,
+      validationState,
```

Comment:
isValid?

Reply:
This is provided by react-aria

Reply:
We can always expose the variable that matches our api pattern and mutate it ourselves to give react-aria the value it needs. I've made the change and approved the PR, feel free to merge.